web-server/v0.4/src/utils/parse.js

```diff
@@ -77,9 +78,17 @@ export const parseIterationData = results => {
           }
 
           iterationData.forEach(hostMetadata => {
-            const columnHost = `client_hostname:${hostMetadata.client_hostname}-server_hostname:${
-              hostMetadata.server_hostname
-            }-server_port:${hostMetadata.server_port}`;
+            const clientHostname = hostMetadata.client_hostname
+              ? `client_hostname:${hostMetadata.client_hostname}`
+              : ``;
+            const serverHostname = hostMetadata.server_hostname
+              ? `server_hostname:${hostMetadata.server_hostname}`
+              : ``;
+            const serverPort = hostMetadata.server_port
+              ? `server_port:${hostMetadata.server_port}`
+              : ``;
+
+            const columnHost = `${clientHostname}${serverHostname}${serverPort}`;
```

Comment:
Why does it work to mash up these names without a separator?

Reply:
This works because `columnHost` is also assigned as the dataIndex for the table. However, we should include separators here for better readability.